icu4c/source/i18n/unicode/uspoof.h

```diff
@@ -359,6 +359,45 @@
  * COMMON or INHERITED, such as numbers and punctuation, are ignored when computing whether a string has multiple
  * scripts.
  *
+ * <h2>Advanced bidirectional usage</h2>
+ * If the paragraph direction with which the identifiers will be displayed is not known, there are
+ * multiple options for confusable detection depending on the circumstances.
+ *
+ * <p>
+ * In some circumstances, the only concern is confusion between identifiers displayed with the same
+ * paragraph direction.  For instance, if the identifiers are usernames prefixed with the @ symbol,
+ * that symbol will appear to the left in a left-to-right context, and to the right in a
+ * right-to-left context, so that an identifier displayed in a left-to-right context can never be
+ * confused with an identifier displayed in a right-to-left context.
+ *
+ * <p>
+ * For example, in those circumstances, the usernames "A1א" (A one aleph) and "Aא1" (A aleph 1)
+ * would be considered confusable, since they appear identical in a left-to-right context, and the
+ * usernames "אA_1" (aleph A underscore one) and "א1_A" (aleph one undescore A) would be considered
+ * confusable, since they appear identical in a right-to-left context.
+ * However, the username "Mark_" would not be considered confusable with the username "_Mark",
+ * even though the latter would appear as Mark_@ in a right-to-left context, and the
+ * former as \@Mark_ in a left-to-right context.
+ *
+ * <p>
+ * In that case, the caller should check for either LTR-confusability or RTL-confusability:
```

Comment:
s/either...or/both...and/

The sample code checks for both, not one vs. the other.